test/jdk/java/util/Arrays/Sorting.java

```diff
@@ -26,7 +26,8 @@
  * @compile/module=java.base java/util/SortingHelper.java
  * @bug 6880672 6896573 6899694 6976036 7013585 7018258 8003981 8226297
  * @build Sorting
- * @run main Sorting -shortrun
+ * @run main/othervm -XX:+UnlockDiagnosticVMOptions -XX:DisableIntrinsic=_arraySort,_arrayPartition, Sorting -shortrun
+ * @run main/othervm -XX:CompileThreshold=1 -XX:-TieredCompilation Sorting -shortrun
```

Comment:
It would be useful to target the compilation threshold only for the intrinsic methods, but i suspect that is not possible?

Reply:
Hi Paul,

Did some further study and found that it's possible to scale the compilation threshold only for the intrinsic methods (`sort `and `partition`)  using the `CompileCommand` and `CompileThersholdScaling` 

 ``` 
-XX:-TieredCompilation -XX:CompileCommand=CompileThresholdScaling,java.util.DualPivotQuicksort::sort,0.0001  -XX:CompileCommand=CompileThresholdScaling,java.util.DualPivotQuicksort::partition,0.0001
```

Will push this change along with build script change suggested by Magnus.

Thanks,
Vamsi